project.mml

```diff
@@ -420,10 +420,15 @@ Layer:
       table: |-
         (SELECT
             way,
-            building
+            building,
+            amenity,
+            aeroway
```

Comment:
I thought all the called keys need to be put here, but the code works without these 2, so I removed them and pushed a new version.